### ISSUE TITLE
Is it intentional that v0.6.5 does not support Rails 6?

### ISSUE THREAD

**Issue by owst** [afair/postgresql_cursor#59]
Hi,

Thanks for the recent release. I've just tried to install 0.6.5 but it's not compatible with Rails 6 - was this intentional? We're likely to be a few months away from upgrading to Rails 7 but would really like to keep using this gem. Was the restriction intentional? 

The following minimal Gemfile shows the problem when using `bundle install`:
```ruby
source "https://rubygems.org"

gem "rails", "~> 6.1.7"
gem "postgresql_cursor", "~> 0.6.5"
```

```
Bundler could not find compatible versions for gem "activerecord":
  In Gemfile:
    postgresql_cursor (~> 0.6.5) was resolved to 0.6.5, which depends on
      activerecord (>= 7.0.0)

    rails (~> 6.1.7) was resolved to 6.1.7, which depends on
      activerecord (= 6.1.7)
```

Thanks,
Owen.

**Comment by hlascelles** [comment]
The `Appraisal` config looks like it supports AR as low as 4.2.11.1, so it would be good to align that with the `gemspec` too.  https://github.com/afair/postgresql_cursor/blob/4b6c630b3f3ae6199258036fab3428a828ea5bc8/Appraisals

**Comment by afair** [comment]
Sorry about that! I mistakenly set the minimum activerecord version to 7. Fixed and released v0.6.6
